hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestSyncTimeRangeTracker.java

```diff
@@ -84,23 +86,23 @@ public void run() {
     assertTrue(trr.getMin() == 0);
   }
 
-  static class RandomTestData {
-    private long[] keys = new long[NUM_KEYS];
-    private long min = Long.MAX_VALUE;
-    private long max = 0;
+ static class RandomTestData {
+    private final AtomicLongArray keys = new AtomicLongArray(NUM_KEYS);
+    private long min = Long.MAX_VALUE; // effectively final
+    private long max = 0; // effectively final
```

Comment:
Don't want to make them final?

Reply:
they get assigned like right below here.

Reply:
Right, would be ideal to make them final, settling for marking them effectively final.